src/test/java/io/confluent/examples/streams/MapFunctionLambdaIntegrationTest.java

```diff
@@ -33,7 +33,7 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 /**
- * End-to-end integration test based on MapFunctionLambdaExample, using an embedded Kafka cluster.
+ * End-to-end integration test based on MapFunctionLambdaExample, using TopologyTestDriver.
```

Comment:
I guess it's not an end-to-end test at all (and never was). Maybe the Javadoc doesn't have to state things that are readily apparent from the code anyway.

Reply:
Renamed to "Test based on MapFunctionLambdaExample, using TopologyTestDriver."